dds/DCPS/XTypes/TypeLookupService.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#include "DCPS/DdsDcps_pch.h"
+#include "TypeLookupService.h"
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace XTypes {
+
+TypeLookupService::TypeLookupService()
+{
+  to_empty_.minimal.kind = TK_NONE;
+  to_empty_.complete.kind = TK_NONE;
+  rpc_sequence_number_ = 0;
```

Comment:
Use constructor's member initializer list

Reply:
Is it possible for `to_empty_.minimal.kind = TK_NONE;`?

Reply:
No, I'm only "pointing to" the single line.  GitHub does a terrible job of rendering it...